### ISSUE TITLE
rules_rust warnings after upgrading to 0.0.40

### ISSUE THREAD

**Issue by jjudd** [dzbarsky/rules_rs#51]
I'm converting our main monorepo to use `rules_rs`. Thanks for your work on it. Excited to use it.

I got things working on 0.0.39 without warnings, then I upgraded to 0.0.40 and noticed these warnings when building Rust targets:

```
warning: variant `Terminate` is never constructed
  --> external/rules_rs++rules_rust+rules_rust/util/process_wrapper/output.rs:29:5
   |
26 | pub(crate) enum LineOutput {
   |                 ---------- variant in this enum
...
29 |     Terminate,
   |     ^^^^^^^^^
   |
   = note: `LineOutput` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis
   = note: `#[warn(dead_code)]` (part of `#[warn(unused)]`) on by default

warning: field `0` is never read
  --> external/rules_rs++rules_rust+rules_rust/util/process_wrapper/rustc.rs:42:10
   |
42 |     Emit(String),
   |     ---- ^^^^^^
   |     |
   |     field in this variant
   |
   = note: `RustcMessage` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis
help: consider changing the field to be of unit type to suppress this warning while preserving the field numbering, or remove the field
   |
42 -     Emit(String),
42 +     Emit(()),
   |

warning: 2 warnings emitted
```

I believe this might be related to this patch: https://github.com/dzbarsky/rules_rust/commit/c9afee2e241042dca4cbabd2e8e6a5a27f3a7428
